stories/general.css

```diff
@@ -32,7 +32,7 @@ input[type='number'] {
 }
 
 .editorFieldContainer > :first-child {
-  color: white;
+  color: rgb(172, 172, 172);
```

Comment:
I had made this white so it would stand out as the header.  We should add bold and possibly underline to this to make it stand out.